From f81a246839b459c57ed89e2d267ae7e07c8c73c1 Mon Sep 17 00:00:00 2001
From: David Faure <faure@kde.org>
Date: Fri, 15 Feb 2019 16:24:07 +0100
Subject: Fix copy/paste typo on variable name

Detected by this Qt warning showing up:
QObject::connect: Cannot connect (null)::checkModifiedFinished() to SvnActions::checkUpdateThread()

CCMAIL: Ch.Ehrlicher@gmx.de
---
 src/svnfrontend/svnactions.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/svnfrontend/svnactions.cpp b/src/svnfrontend/svnactions.cpp
index 0f27fed..9a5b4e3 100644
--- a/src/svnfrontend/svnactions.cpp
+++ b/src/svnfrontend/svnactions.cpp
@@ -2675,7 +2675,7 @@ bool SvnActions::createUpdateCache(const QString &what)
         return false;
     }
     m_UThread = new CheckModifiedThread(this, what, true);
-    connect(m_CThread, SIGNAL(checkModifiedFinished()),
+    connect(m_UThread, SIGNAL(checkModifiedFinished()),
             this, SLOT(checkUpdateThread()));
     m_UThread->start();
     emit sigExtraStatusMessage(i18n("Checking for updates started in background"));
-- 
cgit v1.1
